setup.py

```diff
@@ -18,7 +18,7 @@
                         extra_link_args=["-std=c++11"])]
 
 setup(name='autode',
-      version='1.0.4',
+      version='1.1.0dev',
```

Comment:
should this stay as dev if its being merged?

Reply:
It should be being merged into https://github.com/duartegroup/autodE/tree/v1.1dev so ok?

Reply:
ah right, definitely then :)